docs/reference/release-policy.md

```diff
@@ -3,72 +3,110 @@ id: release-policy
+
+- Customers are automatically updated to the latest component version when it is ready for release.
+- Admins can [enable alpha features](/components/console/manage-organization/enable-alpha-features.md) for non-cluster components in organization settings.
+
+### New Camunda 8 versions
+
+When a new Camunda 8 version is released, we try to provide the new version on our managed service at the same time.
+
+An **Update available** notification is shown in Console, recommending that you update to the latest version.
 
 ![Console with notice to update the cluster in Camunda 8 SaaS](img/update-console.png)
 
-With the Camunda 8.5.0 release, the generation naming scheme in Camunda 8 SaaS will change and no longer include the patch version.
+As of Camunda 8.5.0, the generation naming scheme in Camunda 8 SaaS changed to no longer include the patch version.
 
-The new naming scheme used for the Camunda 8.5 generations will be `Camunda <Major>.<Minor>+gen<N>`, where `N` is incremented with every atomic change to the component version set.
+- The naming scheme used for the Camunda 8.5 generations is `Camunda <Major>.<Minor>+gen<N>`, where `N` is incremented with every atomic change to the component version set.
 
-This was done to decouple the generation name from the particular patch level of the components it contains, as some component versions like Connectors are decoupled from other components.
+- This decouples the generation name from the particular patch level of the components it contains, as some component versions like Connectors are decoupled from other components.
 
-You will learn about the particular component patch version changes in the update dialogue to the latest generation available.
+- You can learn about the particular component patch version changes in the update dialogue to the latest generation available.
 
-#### Updates or restart for critical issues
+#### Update or restart for critical issues
 
 In our managed service, we reserve the right to force update or restart a cluster immediately and without notice in advance if there is a critical security or stability issue.
 
 ## Self-Managed
 
-Whenever a new Camunda 8 version is released, Camunda 8 Self-Managed enterprise customers will be notified via email.
+When a new Camunda 8 version is released, Camunda 8 Self-Managed enterprise customers are notified via email.
 
-If you are not an enterprise customer, you can stay up to date via [release blogs](https://camunda.com/blog/category/release-notes/), the [announcements page](/reference/announcements.md), or releases on [GitHub](https://github.com/camunda) and [Docker Hub](https://hub.docker.com/u/camunda).
+:::info
+Non-enterprise customers can stay up to date via [release blogs](https://camunda.com/blog/category/release-notes.md), [announcements](/reference/announcements.md), or releases on [GitHub](https://github.com/camunda) and [Docker Hub](https://hub.docker.com/u/camunda).
```

Comment:
Link https://docs.camunda.io/docs/next/reference/release-notes/?

Reply:
Changed to https://camunda.com/blog/category/releases/